js/app.js

```diff
@@ -0,0 +1,9 @@
+angular.module('memberApp', [])
+    .controller('MemberListController', ['$scope', '$http',
```

Comment:
Better to use $resource - it uses $http underneath but is better setup for working with a REST API